recipes/_install-windows.rb

```diff
@@ -24,17 +24,19 @@
 temp_file = ::File.join(Chef::Config[:file_cache_path], 'ddagent-cli.msi')
 
 # Download the installer to a temp location
-remote_file 'MSI installer' do
-  path temp_file
+remote_file temp_file do
   source node['datadog']['windows_agent_url'] + dd_agent_msi
+  checksum node['datadog']['windows_agent_checksum'] if node['datadog']['windows_agent_checksum']
   # As of v1.37, the windows cookbook doesn't upgrade the package if a newer version is downloaded
   # As a workaround uninstall the package first if a new MSI is downloaded
   notifies :remove, 'windows_package[Datadog Agent]', :immediately
 end
 
 # Install the package
-windows_package 'Datadog Agent' do
+windows_package 'Datadog Agent' do # ~FC009
```

Comment:
@bigbam505 can you please clarify what resource attribute is not being recognized here?